diagnostic.md

```diff
@@ -45,7 +61,13 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+```CSS
+.big{}
+#contentPane{}
+.important, .red{}
```

Comment:
This will select everything that has either ```important``` or ```red``` as a class.  To select elements that have both ```important``` and ```red``` as classes, use ```.important.red```.